README.md

```diff
@@ -22,12 +25,7 @@ you write better policies and have fun while doing it!
 - Identify common mistakes, bugs and inefficiencies in Rego policies, and suggest better approaches
 - Provide advice on [best practices](https://github.com/StyraInc/rego-style-guide), coding style, and tooling
 - Allow users, teams and organizations to enforce custom rules on their policy code
-
-Regal rules are to as large extent as possible
-[written in Rego](https://www.styra.com/blog/linting-rego-with-rego/) themselves,
-using the JSON representation of the Rego abstract syntax tree (AST) as input, a
-few additional custom built-in functions and some indexed data structures to help
-with linting.
+- Deliver an outstanding policy development experience by providing the best possible tools for that purpose
```

Comment:
Move to the top of Goals, perhaps? Nobody reads the last goal :P

Reply:
Fair enough :)